src/gc.c

```diff
@@ -644,6 +644,37 @@ void gc_setmark_buf(jl_ptls_t ptls, void *o, uint8_t mark_mode, size_t minsz)
     gc_setmark_buf_(ptls, o, mark_mode, minsz);
 }
 
+void jl_gc_force_mark_old(jl_ptls_t ptls, jl_value_t *v)
```

Comment:
This (and `gc_sweep_sysimg` etc below) should go into a header. It happens way too often when we put declaration in C files and they get out of sync when the implementation is updated.